docs/Technical Reference.md

```diff
@@ -2,110 +2,102 @@ This page provides a technical reference on certain advanced configuration optio
 
 ## Deploying the T4 Catalog on AWS
 
-The following section uses AWS CloudFormation to deploy and manage the resources required to run a T4 bucket and catalog in your own VPC.
+The following instructions use CloudFormation to install T4 on a bucket in your AWS account.
 
+<!--
 ### Known limitations
 
 * Supports only one bucket
 * Search is only enabled for *new objects* uploaded through T4's Python API
 * Any IAM users with `ESFullAccess` or `AdministratorAccess` can invoke ElasticSearch over your bucket
+-->
 
-### Pre-requisites
+1. Ensure you have sufficient permissions to proceed. The `AdministratorAccess` policy is sufficient.
+2. If you are going to use Quilt T4 with an existing bucket, make sure that your target bucket has [object versioning enabled](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/enable-versioning.html), as well as the following [CORS policy](https://docs.aws.amazon.com/AmazonS3/latest/dev/cors.html#how-do-i-enable-cors):
 
-1. *Sufficient permissions* - Creating a CloudFormation stack requires sufficient permissions, such as the
-`AdministratorAccess` policy
-1. *A properly configured S3 bucket* where you will search, store, and browse data with Quilt. It is **highly recommended** that you back up said S3 bucket before proceeding.
-   * [Enable object versioning](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/enable-versioning.html) on the bucket
-   * Set a [CORS policy](https://docs.aws.amazon.com/AmazonS3/latest/dev/cors.html#how-do-i-enable-cors), similar to the following, on the bucket:
+    ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
+    <CORSRule>
+        <AllowedOrigin>https://yourcompanydomain.com</AllowedOrigin>
```

Comment:
Please make it more obvious that this is a parameter the user needs to substitute, I think our convention is all-caps with a `$` at the beginning, so it'd be `https://$YOURCOMPANYDOMAIN.COM`